cmd/cli.go

```diff
@@ -158,6 +158,16 @@ func (cli *CLI) Run(args []string) int {
 	}
 	runners = append(runners, runner)
 
+	var ruleNames []string
+	for name := range cfg.Rules {
+		ruleNames = append(ruleNames, name)
+	}
+	err = rules.CheckRuleNames(ruleNames)
```

Comment:
I'm happy if an equivalent check will be also performed in LSP mode:
https://github.com/wata727/tflint/blob/96dd0d36f38515b8def4d4ccb37ff31da28bcb7c/langserver/handler.go#L22-L35

Reply:
Also done.

How these options are set for LSP?

Reply:
These can be set via `.tflint.hcl` file.